xep-0313.xml

```diff
@@ -16,6 +16,7 @@
   <dependencies>
     <spec>XMPP Core</spec>
     <spec>XEP-0030</spec>
+    <spec>XEP-0045</spec>
```

Comment:
Not sure if we should list XEP-45 as dependency. If at all, it's an optional one. But then you had to list XEP-60 too.